content/videos/challenges/19-superellipse/showcase/contribution2.json

```
@@ -0,0 +1,8 @@
+{
+  "title": "Piet Hein Rainbow",
+  "url": "https://editor.p5js.org/nicolaimg/full/q-kLyOogz",
+  "author": {
+    "name": "Nicolai Munch Grundsøe",
+    "url": "ig: @nicolai.gif"
```

```suggestion
    "url": "https://www.instagram.com/nicolai.gif/"
```

This has to be a URL unfortunately for it to work!